test/conditionals.js

```diff
@@ -369,6 +369,11 @@ describe('Conditionals', function () {
     done()
   })
 
+  it('should evaluate <if doesntexist> as false and trigger <else> condition with embedded HTML comments in conditional statements (conditionals/ifCommentsEmbded.html.html)', function (done) {
```

Comment:
```suggestion
  it.skip('should evaluate <if doesntexist> as false and trigger <else> condition with embedded HTML comments in conditional statements (conditionals/ifCommentsEmbded.html)', function (done) {
```

- marking skip so it doesn't break the build
- you should rename the template to spell it correctly
- also file an issue for this and link to this PR

Reply:
Sorry I was follow previous workflows here, didn't know if you wanted to stage all the changes in this branch. Anyways I did what you asked for